doc/schemas/message.schema.json

```diff
@@ -0,0 +1,44 @@
+{
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "title": "APNS Push Message",
+    "description": "A Push message for APNS",
+    "type": "object",
+    "properties": {
```

Comment:
I think we should set `additionalProperties` to `false` here, or however it's called and wherever it's applicable. The setting to state that not-mentioned properties are not allowed.

Reply:
I don't think we can do that on the top level because of the custom properties, but I'll find out where I can add it.